### PR TITLE
[ZH] Add `How you can help!`

### PR DESCRIPTION
**Pull request by FlyingFish23**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

麻烦reviewer在提出修改建议的时候给个理由哦
please leave a reason for me to check when suggesting changes! thanks~
然后鸡蛋里挑骨头，为了改而改的就别提了，我会直接resolve的
please, don't try to pick bones from an egg, or suggest changes just because you can. I'll just ignore and resolve them
我是学文的，你觉得我遣词造句有问题可以拿新华字典解释或者拿相关guideline来怼我，别无中生有哈

### CODE REVIEW

File: wiki/Community/How_you_can_help!/zh.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  - [皮肤制作](https://osu.ppy.sh/community/forums/15)
+  - [游戏与排名](https://osu.ppy.sh/community/forums/13)
+- 阅读板块的置顶帖，来了解到板块相关的规则和其他有用的信息。
+- 使用顶端的 `Mark forum as read` 按钮。
+- 关注新帖，有空的话就阅读一下并且提供力所能及的帮助 —— 你可以使用未读图标来确保自己跟上了进度。
+- 读一读[可用的指南](/wiki/Guides)来扩展自己的 osu! 知识面，如果里面的内容不能满足你，那么欢迎你尝试参与贡献！
+
+## 帮助检查或摸图！
+
+osu! 有专注于检查新提交[谱面](/wiki/Beatmap)是否能够 Rank 并被所有人游玩的团队，[谱面审核成员 (BN)](/wiki/People/The_Team/Beatmap_Nominators)。但这不意味着你不能在这一领域发光发热：
```
Comment (heipizhu4) [inline wiki/Community/How_you_can_help!/zh.md:45]: 很喜欢这句的翻译。

**Review by heipizhu4 — APPROVED** [review]

File: wiki/Community/How_you_can_help!/zh.md (hunk 1 above)
Comment (FlyingFish23) [reply wiki/Community/How_you_can_help!/zh.md:45]: 谢谢你的认可☺️

**Comment by heipizhu4** [comment]
Ah, you forgot to tick the box at the head of the PR. Tick and get merged!

**Comment by FlyingFish23** [comment]
> Ah, you forgot to tick the box at the head of the PR. Tick and get merged!

Done, just ticked it!

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**